src/plugins/intel_gpu/src/plugin/common_utils.cpp

```diff
@@ -138,6 +138,20 @@ void convert_and_copy(const cldnn::memory::ptr src, ov::ITensor const* dst, cons
     return ::convert_and_copy(src_ptr, src_et, dst_ptr, dst_et, size, src->get_layout());
 }
 
+void convert_and_copy(const cldnn::memory::ptr src, cldnn::memory::ptr dst, cldnn::stream& stream) {
+    const bool blocking = false;
```

Comment:
Please make this call blocking, because otherwise you need to process event properly